### PR TITLE
Add Concord Configuration

### PR DESCRIPTION
**Pull request by MRHarrison**
- Adding configuration in order to get concord working

### CODE REVIEW

File: supply-chain/src/environments/environment.concord-dev.ts [hunk 1]
```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright 2019 VMware, all rights reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+export const environment = {
+  production: false,
+  blockchainType: 'ganache',
```
Comment (mandeabhi) [inline supply-chain/src/environments/environment.concord-dev.ts]: Would the blockchainType be 'concord' here?

Comment (MRHarrison) [reply supply-chain/src/environments/environment.concord-dev.ts]: I can make this more generic.  And call it no auth.  Basically this parameter is designed to be used for which type `web3` initialization to use, concord and ganache have a very similar `web3` initialization.

Comment (MRHarrison) [reply supply-chain/src/environments/environment.concord-dev.ts]: But yeah I'll switch it to concord, probably useful later for minor differences

Comment (MRHarrison) [reply supply-chain/src/environments/environment.concord-dev.ts]: Added

**Pull request merged**